clients/src/main/java/org/apache/kafka/clients/consumer/internals/CommitRequestManager.java

```diff
@@ -760,13 +771,22 @@ public void onResponse(final ClientResponse response) {
             boolean failedRequestRegistered = false;
             for (OffsetCommitResponseData.OffsetCommitResponseTopic topic : commitResponse.data().topics()) {
                 for (OffsetCommitResponseData.OffsetCommitResponsePartition partition : topic.partitions()) {
-                    TopicPartition tp = new TopicPartition(topic.name(), partition.partitionIndex());
+                    // Version 10 drop topic name and support to topic id.
```

Comment:
This doesn't read very well, we could maybe rephrase it to  
```suggestion
                    // Version 10 drops topic name, and supports topic id.
```